airflow/providers/apache/kafka/operators/consume.py

```diff
@@ -0,0 +1,184 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+from __future__ import annotations
+
+from functools import partial
+from typing import Any, Callable, Sequence
+
+from airflow.exceptions import AirflowException
+from airflow.models import BaseOperator
+from airflow.providers.apache.kafka.hooks.consume import KafkaConsumerHook
+from airflow.utils.module_loading import import_string
+
+VALID_COMMIT_CADENCE = {"never", "end_of_batch", "end_of_operator"}
```

Comment:
is it private or public set? and I see it defined in the other operators/sensors, we may define it in a utils module. Also it looks more like an Enum.
WDYT?

Reply:
Yeah this was on my radar and something we can do it the future.

Reply:
> As there's no such thing as a private variable in python i'm not 100% sure what you're asking.

If this variable is intended for internal use only, it's a good practice to add a single underscore before its name. This convention signals to the developers that the variable is intended for internal use and should not be accessed from outside the module. Additionally, when using the `from module import *` syntax to import all variables from the module, variables with a single underscore prefix will not be imported.

> re: Enum. Yeah this was on my radar and something we can do it the future.

okay